accounts/.drone.star

```diff
@@ -633,8 +620,8 @@ def manifest(ctx):
     ],
     'depends_on': [
       'amd64',
-      'arm64',
-      'arm',
+      #'arm64',
+      #'arm',
```

Comment:
Why disabling ARM builds?

Reply:
@phil-davis my exact thoughts. But this is the current status on master: https://github.com/owncloud/ocis-accounts/blob/master/.drone.star#L9-L10

Depending on how "pure" we want this PR to be, we could either merge as is (reflecting changes to accounts-master 1-to-1) or do changes on this branch.

Opinions @micbar ?

Reply:
this is the old accounts .drone.star, irrelevat for the ocis monorepo